modules/core/src/shadertools/src/utils/webgl-info.js

```diff
@@ -88,7 +88,8 @@ export function canCompileGLGSExtension(gl, cap, opts = {}) {
   }
 
   const extensionName = feature[0];
-  const source = `#extension GL_${extensionName} : enable\nvoid main(void) {}`;
+  const behavior = opts.behavior || 'enable';
```

Comment:
Add documentation of `opts` for this function

Reply:
Done.